tasks/main.yml

```diff
@@ -55,29 +55,48 @@
     dest: "{{ golem_source_path }}/golem_git"
     version: "{{ golem_source_version }}"
 
+- name: Add python 3.6 repository apt key
+  apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: 4ab0f789cba31744cc7da76a8cf63ad3f06fc659
+  become: yes
+- name: Add python 3.6 repo
```

Comment:
add an empty line between tasks

Reply:
Added here and checked the rest of the file :)